### PR TITLE
Add note about multipart features

### PR DESCRIPTION
**Pull request by geographika**
Was attempting to do this using -99 "magic" numbers as per symbols

### CODE REVIEW

File: en/mapfile/feature.txt [hunk 1]
```diff
@@ -21,6 +21,15 @@ POINTS
     
        POLYGON/POLYLINE layers POINTS must start and end with the 
        same point (i.e. close the feature).
+       
+    Multipart features can be created by adding further sets of points, for example:
+    
+    ::
+    
+	  FEATURE
+         POINTS 1 1 50 50 1 50 1 1 END
```
Comment (yjacolin) [inline en/mapfile/feature.txt]: There is indentation issue here? Something like this:
```
FEATURE
         POINTS 1 1 50 50 1 50 1 1 END
         ...
END
```

**Comment by jmckenna** [comment]
oops sorry missed @yjacolin comments!

**Comment by yjacolin** [comment]
no problem @jmckenna I was ready to Approve it even with my comments.

**Comment by geographika** [comment]
Apologies - was using the online editor, and mixed tabs and spaces. Thanks all.

**Pull request merged**